### PR TITLE
add: about page cleanup and icons navbar link

### PR DESCRIPTION
**Pull request by simonhlee97**
* a link to About in navbar
* new layout and cards for contributors
* github icons

### CODE REVIEW

File: apps/web/src/pages/About/index.tsx [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import Members from '@/components/Members';
 import { StyledAbout } from './styles';
 
+// const About = () => (
```
Comment (ivanms1) [inline apps/web/src/pages/About/index.tsx:4]: Please remove commented code

**Review by ivanms1 — COMMENTED** [review]
great job!
is it mobile friendly?

File: apps/web/src/pages/About/index.tsx (hunk 1 above)
Comment (simonhlee97) [reply apps/web/src/pages/About/index.tsx]: okay, done.

File: apps/web/src/pages/Home/index.tsx [hunk 2]
```diff
@@ -33,53 +32,55 @@ function Home() {
...
-              mollitia natus eveniet. Hic.
-            </p>
-            <Button>Sign up</Button>
-          </StyledContentBox>
-          <Image
-            src={'/assets/images/shelf.png'}
-            alt='project shelf logo'
-            height={400}
-            width={400}
+    <>
```
Comment (ivanms1) [inline apps/web/src/pages/Home/index.tsx:35]: why was this fragment added, you can remove it I think

**Review by ivanms1 — APPROVED** [review]
LGTM, just check my small comment please

**Pull request merged**